selfservice/strategy/password/validator_test.go

```diff
@@ -30,6 +50,12 @@ func TestDefaultPasswordValidationStrategy(t *testing.T) {
 		{pw: "l3f9toh1uaf81n21", id: "l3f9toh1uaf81n21", pass: false},
 		{pw: "l3f9toh1", pass: true},
 		{pw: "000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000", pass: true},
+		// simple permutation tests
+		{id: "hello@example.com", pw: "hello@example.com12345", pass: false},
+		{id: "hello@example.com", pw: "123hello@example.com123", pass: false},
+		{id: "hello@example.com", pw: "hello@exam", pass: false},
+		{id: "hello@example.com", pw: "HELLO@EXAMPLE.COM", pass: false},
```

Comment:
Let's add more test cases:

```suggestion
		{id: "hello@example.com", pw: "HELLO@EXAMPLE.COM", pass: false},
		{id: "hello@example.com", pw: "HELLO@example.com", pass: false},
		{pw: "hello@example.com", id: "hello@exam", pass: false},
		{id: "hello@example.com", pw: "h3ll0@example", pass: false},
		{pw: "hello@example.com", id: "hello@exam", pass: false},
		{pw: "abcd", id: "9d3c8a1b", pass: true},
		{pw: "a", id: "kjOkla", pass: true},
```